sherpa/utils/__init__.py

```diff
@@ -2616,6 +2605,8 @@ def func_counter(func):
     warnings.warn("func_counter is deprecated in 4.17.0: use FuncCounter instead",
                   FutureWarning)
 
+    nfev = [0]
+
```

Comment:
Why line 2608 was added? is this to fix a bug?

Reply:
Yes, as mentioned

- in the details section above
- in the commit log that made the change: https://github.com/sherpa/sherpa/pull/2290/commits/fd0ddbf1dd38b5300bc20302e8dffd810208f96d